pkg/nsx/services/securitypolicy/builder_test.go

```diff
@@ -231,10 +231,6 @@ func TestBuildTargetTags(t *testing.T) {
 					Scope: &tagScopeNamespace,
 					Tag:   &tagValueNS,
 				},
-				{
```

Comment:
why removed this tag?

Reply:
oh, refactored BuildBasicTags and made a bug here, fixed, Thanks~~